test/src/tests-main.js

```diff
@@ -19,7 +19,7 @@ beforeEach(function() {
         requestConfig: false,
         isDevelopmentMode: false,
         flags: {
-            eventBatchingIntervalMillis: 0,
+            eventBatchingIntervalMillis: "0",
```

Comment:
Why does this need to be a string now?

Reply:
It's returned this way from the server, so we just need to come to a consensus for when it's turned into a number.